ATP.BusinessLogicLayerTest.cs/ServicesTest/ManagerServiceTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using ATP.BusinessLogicLayer.Models;
+using ATP.BusinessLogicLayer.Services;
+namespace ATP.BusinessLogicLayerTest.ServicesTest;
+
+public class ManagerServiceTest
+{
+    private readonly ManagerService _managerService;
+    public ManagerServiceTest()
+    {
+        var flights = new List<FlightDomainModel>
+        {
+            new FlightDomainModel(1, 100.0, "CountryA", "CountryB", DateTime.Now, "AirportA", "AirportB", FlightClass.Economy),
+            new FlightDomainModel(2, 150.0, "CountryA", "CountryC", DateTime.Now, "AirportA", "AirportC", FlightClass.Business),
+            new FlightDomainModel(3, 200.0, "CountryD", "CountryB", DateTime.Now, "AirportD", "AirportB", FlightClass.First)
+        };
+        _managerService = new ManagerService(flights);
+    }
+
+    [Fact]
+    public void FilterByFlight_ValidInput_ReturnsFilteredFlights()
```

Comment:
this testing many many scenarios, it must be one scenario. 
Create 4 tests like:

1.  FilterByFlight_ShouldReturnFlight_WhenDepartureAndDestinationMatch
2. FilterByFlight_ShouldReturnEmpty_WhenDepartureCountryNotFound
3. FilterByFlight_ShouldReturnEmpty_WhenDestinationCountryNotFound
4. FilterByFlight_ShouldBeCaseInsensitive